### PR TITLE
Update to use with Prebid.js v1

### PR DESCRIPTION
**Pull request by felipemsantana**

### CODE REVIEW

File: src/_prebid.jsx [hunk 1]
```diff
@@ -14,24 +14,39 @@ const propTypes = {
...
+    PropTypes.oneOf(['low', 'medium', 'high', 'auto', 'dense']),
+    PropTypes.shape({
+      buckets: PropTypes.arrayOf(PropTypes.shape({
+        min: PropTypes.number,
+        max: PropTypes.number,
+        increment: PropTypes.number,
+        precision: PropTypes.number,
+      })),
+    }),
+  ]),
```
Comment (rafaelverger) [inline src/_prebid.jsx:41]: I think this is wrong... priceGranularity could be any of those strings in that array (line#30) or the object with that shape (line#31):
```js
priceGranularity: PropTypes.oneOf([
  'low', 'medium', 'high', 'auto', 'dense',
  PropTypes.shape({
    buckets: PropTypes.arrayOf(PropTypes.shape({
      min: PropTypes.number,
      max: PropTypes.number,
      increment: PropTypes.number,
      precision: PropTypes.number,
    })),
  }),
]);
```

File: src/_prebid.jsx [hunk 2]
```diff
@@ -47,10 +62,15 @@ export default class PrebidContainer extends Component {
   }
 
   configure() {
-    const { config } = this.props;
-    if (config.pricing) {
-      pbjs.setPriceGranularity({ buckets: config.pricing });
-    }
+    const { bidderTimeout, currency, debug, priceGranularity } = this.props;
+    // The pbjs.setConfig param object must be JSON
```
Comment (rafaelverger) [inline src/_prebid.jsx]: All those props area already validated by propTypes, why are u dumping and parsing it?

File: src/_prebid.jsx [hunk 3]
```diff
@@ -62,10 +82,6 @@ export default class PrebidContainer extends Component {
     if (!window.pbjs) {
       window.pbjs = { que: [this.configure] };
       window.pbjs__slots = [];
-      window.pbjsAdServerTimeout = setTimeout(() => {
-        console.log('[prebid] global timeout');
-        this.sendAdserverRequest();
-      }, this.props.adserverTimeout);
```
Comment (rafaelverger) [inline src/_prebid.jsx:68]: `bidsBackHandler` is called when `bidderTimeout` is threw?

**Review by rafaelverger — CHANGES_REQUESTED** [review]

File: src/_prebid.jsx (hunk 1 above)
Comment (felipemsantana) [reply src/_prebid.jsx:41]: `PropTypes.oneOf` is for constants, this way is telling it is allowed those strings or `PropTypes.shape...` as a value

File: src/_prebid.jsx (hunk 2 above)
Comment (felipemsantana) [reply src/_prebid.jsx]: They must be JSON, as stated in the docs, `undefined` is not a valid JSON value, the lib uses `for` to iterate the config properties and calling some functions, passing `undefined` it throws errors

File: src/_prebid.jsx (hunk 3 above)
Comment (felipemsantana) [reply src/_prebid.jsx:68]: http://prebid.org/dev-docs/publisher-api-reference.html#pbjsrequestbidsrequestobj
>Callback to execute when all the bid responses are back or the timeout hits.

File: src/_prebid.jsx (hunk 2 above)
Comment (rafaelverger) [reply src/_prebid.jsx]: damm... I would rather use a lodash function to do this cleanup

**Review by rafaelverger — APPROVED** [review]

**Pull request merged**